ui/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
```

Comment:
I am not fond of 100-lines gitignores.

Can we just keep it to the thing we actually need?

Reply:
yes I wiil keep only what I need.